express/extension.js

```diff
@@ -103,4 +109,5 @@ const extension = new Extension();
 
 module.exports = {
     extension
-};
\ No newline at end of file
+};
+const SessionStorage = require('./session/session_storage');
```

Comment:
Move this import to top.

Reply:
Moved down here to solve cyclic dependancy issue.